### DISCUSSION TITLE
How to update hydro?

### DISCUSSION THREAD

**Discussion by junukwon7** [hydro-dev/Hydro#51] [Q&A] [answered]
Hello! I was searching through the page and found that many changes were made. However, I also found that using ```yarn global upgrade-interactive``` is now not provided in the guide page. Was there a major change?
If using it is not recommended, what else could be done in order to make hydro up-to-date?

**Comment by undefined-moe** [comment, marked as answer]
it is supported and suggested. @Macesuted please update the docs
